test/test_helper.rb

```diff
@@ -134,14 +134,17 @@ def ==(other)
 end
 
 class ActionMailer::TestCase
-  def assert_email(email, to, subject, fixture)
+  def assert_email(email, to, subject, fixture, bulk: false)
     # Test email can send ok
     assert_emails 1 do
       email.deliver_now
     end
 
+    tld = bulk ? "io" : "org"
+    from = "hello@mail.exercism.#{tld}"
```

Comment:
We seem to be using a different top-level domain for bulk emails vs transactional emails.